tide/wall/DisplayGroupRenderer.cpp

```diff
@@ -151,7 +151,8 @@ void DisplayGroupRenderer::setDisplayGroup( DisplayGroupPtr displayGroup )
     // of the focus context may not always be restored to its original value.
     // See JIRA issue: DISCL-305
```

Comment:
last time I checked it was still the case with Qt 5.4, if it goes away it will be by pure luck...